sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -4599,6 +4599,16 @@ SIREPO.app.service('utilities', function($window, $interval) {
         };
     };
 
+    // create a non-cryptographic baseN string, with N <= 64
+    this.randomString = function(base=62, length=32) {
```

Comment:
YAGNI on specifying the base? Always use base62

Reply:
a4dd0cf